bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -26,25 +26,21 @@ f = h5py.File(args.output_file, "w")
 logging.info('Copying segments and attributes to %s' % args.output_file)
 # Move segments information into the final file - remove some duplication in earlier files
 for fi in files:
-    # Get a list of ifos in the segments of the input data file
-    ifos_in_file = [ifo for ifo in args.ifos if ifo in fi['segments']]
-    # Arbitrarily choose one to copy over the segments for as all are currently the same 
-    ifo_to_get = 'segments/{}'.format(ifos_in_file[0])
-    # Convert list of IFOs into a identifier string
-    ifosstring = ''.join(ifos_in_file)
-    group_to_save = '{}/segments'.format(ifosstring)
-    f[group_to_save + '/end'] = fi[ifo_to_get + '/end'][:]
-    f[group_to_save + '/start'] = fi[ifo_to_get + '/start'][:]
-    # Use this ID string to save the attributes from the input files
+    if len(fi['segments'].keys()) > 1:
+        logging.info('Incorrect segment format.')
+
+    key = fi['segments'].keys()[0]
+    f['%s/segments/end' % key] = fi['segments/%s/end' % key][:]
```

Comment:
This is a point where consistency would be better - we change from segments/key to key/segments

This change was made because of wanting the attributes associated with the groups for each detector combination. These attributes are not necessarily associated with the segments and as a result we went for the combination/segments ordering

I think that having it as combinations/segments instead throughout would make more sense, (but I wouldn't be too upset if others disagree)

Reply:
Please change to use combinations/segments

Reply:
Changed to segments/combinations everywhere